velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -310,6 +326,21 @@ bool canSortInputs(const CallableSignature& signature) {
   return true;
 }
 
+// Returns true if specified aggregate function can be applied to distinct
+// inputs.
+bool supportsDistinctInputs(const CallableSignature& signature) {
+  if (signature.args.size() != 1) {
```

Comment:
signature.args.empty()

We should allow 2+ argument aggregate functions? Why limit to single-arg?